### PR TITLE
   [driver] #1379 - Post Merge Review Update

### PR DESCRIPTION
**Pull request by technicalgrp89**
UPD by @atamazov:
This is expected to resolve https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1282#pullrequestreview-850574173.

----
- Post Merge review comments added.
- Formatting
- Function name & i/p parameter names are changed.

Signed-off-by: Sujatha Phatak <sujatha.phatak@amd.com>

### CODE REVIEW

**Comment by technicalgrp89** [comment]
@atamazov @junliume @JehandadKhan : code review request. Thank you

**Comment by junliume** [comment]
@technicalgrp89  please monitor CI and make sure it passes:
http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/1282_postmerge_update/3/pipeline

File: driver/InputFlags.cpp [hunk 1]
```diff
@@ -55,6 +55,7 @@ void InputFlags::AddInputFlag(const std::string& _long_name,
 
 [[gnu::noreturn]] void InputFlags::Print() const
 {
+    printf("MIOpen Driver Input Flags: \n\n");
```
Comment (junliume) [inline driver/InputFlags.cpp:58]: @atamazov @technicalgrp89 : I'm curious if `printf` can be actually effective here.

Comment (technicalgrp89) [reply driver/InputFlags.cpp:58]: @junliume It was present, before I started working on this change. , so, I retained it. @atamazov

Comment (atamazov) [reply driver/InputFlags.cpp:58]: Yes, this reverts change caught at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1282#discussion_r763403764

**Review by atamazov — DISMISSED** [review]
LGTM!

**Review by junliume — APPROVED** [review]

**Comment by atamazov** [comment]
@technicalgrp89 >  technicalgrp89 added the Review Required label 17 days ago 

Do you really think we need this? The labels that require review are already marked in the list of pull requests, both textually ("Review required") and by a pictogram (:x:). And there is also an "[Awaiting review from me](https://github.com/ROCmSoftwarePlatform/MIOpen/pulls?q=is%3Apr+is%3Aopen+user-review-requested%3A%40me)" list.

**Comment by atamazov** [comment]
@technicalgrp89
> 
> > technicalgrp89 added the Review Required label 17 days ago
> 
> **_Do you really think we need this label?_** The labels that require review are already marked in the list of pull requests, both textually ("_Review required_") and by a pictogram (x). And there is also handy "[Awaiting review from me](https://github.com/ROCmSoftwarePlatform/MIOpen/pulls?q=is%3Apr+is%3Aopen+user-review-requested%3A%40me)" list.

Label removed.

**Pull request merged**